include/effectmanager.h

```diff
@@ -354,11 +354,6 @@ class  EffectManager : public IJSONSerializable
         return _gfx[index];
     }
 
-    size_t gCount()
-    {
-        return _gfx.size();
-    }
-

```

Comment:
This was used in one place, where FastLED.count() was a better alternative.

Reply:
Nice cleanup!